edgelet/iotedge/src/check/checks/up_to_date_config.rs

```diff
@@ -0,0 +1,96 @@
+use std::fs;
+use std::path::Path;
+
+use failure::{self, Context, Fail};
+
+use crate::check::{checker::Checker, Check, CheckResult};
+
+#[derive(Default, serde_derive::Serialize)]
+pub(crate) struct UpToDateConfig {}
+
+impl Checker for UpToDateConfig {
+    fn id(&self) -> &'static str {
+        "aziot-configs-up-to-date"
+    }
+    fn description(&self) -> &'static str {
+        "aziot configurations up-to-date with config.toml"
```

Comment:
I feel we should call it iotedge and not aziot.

Reply:
Fixed.